From d1b95d1676f199dc3bcdc12d94c6c2a7cfd3bb7c Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Mon, 3 Dec 2012 17:59:02 -0600
Subject: [PATCH 03/53] ENGR00235624 Quad/DualLite ARD: MTD partition non
 aligned

MTD partition for SPI-NOR was not aligned to 8K.
Replace its offset from MTDPART_OFS_APPEND to MTDPART_OFS_NXTBLK.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 3142f4f..5b19333 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -417,7 +417,7 @@ static struct mtd_partition m25p32_partitions[] = {
 		.mask_flags = MTD_WRITEABLE,
 	}, {
 		.name	= "kernel",
-		.offset	= MTDPART_OFS_APPEND,
+		.offset	= MTDPART_OFS_NXTBLK,
 		.size	= MTDPART_SIZ_FULL,
 	},
 };
-- 
1.7.10.4

